src/config/index.js

```diff
@@ -12,11 +12,9 @@ import createConfig from 'src/lib/create-config';
 let configFile;
 if ( 'development' === process.env.NODE_ENV ) {
 	configFile = require( 'src/config/development.json' );
-	try {
-		// check if we have local git ignored config and use it to override the development.json
-		const localConfig = require( 'src/config/development.local.json' );
-		merge( configFile, localConfig );
-	} catch ( e ) {}
+	// check if we have local git ignored config and use it to override the development.json
+	const localConfig = require( 'src/config/development.local.json' );
```

Comment:
Instead of creating a fake `development.local.json` file, wouldn't it be better to check whether it exists here?